Dockerfile

```diff
@@ -1,12 +1,9 @@
-FROM node:14.15.4-buster
+FROM node:14.15.4-alpine
```

Comment:
Wonder if this should be `node:14-alpine` or maybe `node:14.15-alpine` so we don't have to bother as much about updates? But being specific is also fine.